index.js

```diff
@@ -194,6 +196,14 @@ function createLocationModel (opts) {
       }
     }
 
+    if (opts.hash !== false) {
+      subs.handleHash = function (send, done) {
```

Comment:
I've been thinking about this: this only handles `window.hash = "#something"` API calls, and I'm not sure we should encourage that interface. If some component needs to change the route it can already do so by passing `send('location:set', '#something')`.

If for some reason something deeper down the tree produces a side effect using `window.hash =` I think we should consider that to be bad practice and encourage developers to not provide side effects like that. Does that make sense?